<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrateSpectrum</class>
 <widget class="QWidget" name="CalibrateSpectrum">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calibrate</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="spectrum_plot_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="calibration_points">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>New Calibration Point</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>x-axis</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Wavelength (Å)</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Dispersion (Å/pix)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="5">
           <widget class="QCheckBox" name="point_is_star">
            <property name="text">
             <string>Zero (star)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QPushButton" name="remove_calibration_point">
            <property name="text">
             <string>Remove selected point</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/resources.qrc">
              <normaloff>:/remove_16.png</normaloff>:/remove_16.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QPushButton" name="wavelength_pick">
            <property name="text">
             <string>Pick</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QPushButton" name="x_axis_pick">
            <property name="text">
             <string>Pick</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="4">
           <widget class="QPushButton" name="set_dispersion">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Set dispersion</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="3">
           <widget class="QDoubleSpinBox" name="dispersion">
            <property name="decimals">
             <number>4</number>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QPushButton" name="add_calibration_point">
            <property name="text">
             <string>Add calibration point</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/resources.qrc">
              <normaloff>:/add_16.png</normaloff>:/add_16.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="3">
           <widget class="QDoubleSpinBox" name="point_wavelength">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100000000000000004764729344.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QSpinBox" name="point_x_axis">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>99999999</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
